ocamldebug.el

```diff
@@ -42,6 +42,7 @@
                   "tuareg" (file-name-directory (or load-file-name
                                                     byte-compile-current-file))))
 (require 'derived)
+(require 'seq)
```

Comment:
I'm not sure if `seq.el` was available in Emacs 24.4 (which `tuareg` currently targets). Perhaps it should be added an explicit package dep?

Reply:
We've since bumped the required Emacs version so it's not an issue any more